test-suite/coq-makefile/timing/run.sh

```diff
@@ -73,12 +73,12 @@ cat A.v.timing.diff
 echo
 
 for ext in "" .desired; do
-    for file in A.v.timing.diff; do
-        cat ${file}${ext} | sed s"/${INFINITY}/${INFINITY_REPLACEMENT}/g" | sed s":|\s*N/A\s*$:| ${INFINITY_REPLACEMENT}:g" | sed s'/[0-9]*\.[0-9]*//g' | sed s'/0//g' | sed s'/  */ /g' | sed s'/ *$//g' | sed s'/+/-/g' | sort > ${file}${ext}.processed
+    for file in A.v.timing.diff/*; do
```

Comment:
This change is not correct, do `file=A.v.timing.diff` removing the loop instead.